### PR TITLE
Add Unstake Tokens Ui

### PR DESCRIPTION
**Pull request by NikolasStelmakh**

### CODE REVIEW

File: client/src/components/Input/StakeAmountInput.tsx [hunk 1]
```diff
@@ -42,8 +42,8 @@ export const StakeAmountInput = (
     input,
     meta,
     availableTitle = 'Available FIO Balance',
-    availableValue = 0,
-    maxValue = availableValue,
+    availableValue = '0',
+    maxValue = +availableValue,
```
Comment (andreyvEze) [inline client/src/components/Input/StakeAmountInput.tsx:46]: maybe they should have the same type

File: client/src/components/common/TransactionResults/Results.tsx [hunk 2]
```diff
@@ -108,6 +109,9 @@ const Results: React.FC<ResultsContainerProps> = props => {
             {totalCost()}
             {totalBundlesCost()}
             <SubmitButton onClick={onClose} text="Close" withTopMargin={true} />
+            {additionalSubmitButtonProps ? (
+              <SubmitButton {...additionalSubmitButtonProps} />
+            ) : null}
```
Comment (andreyvEze) [inline client/src/components/common/TransactionResults/Results.tsx:114]: could we use `bottomElement` instead?

File: client/src/components/common/TransactionResults/components/StakeTokensResults.tsx [hunk 3]
```diff
@@ -10,6 +10,7 @@ import { FIO_CHAIN_CODE } from '../../../../constants/fio';
 import { ResultsProps } from '../types';
 
 import classes from '../styles/Results.module.scss';
+import InfoBadge from '../../../InfoBadge/InfoBadge';
```
Comment (andreyvEze) [inline client/src/components/common/TransactionResults/components/StakeTokensResults.tsx:13]: move to components import

File: client/src/components/common/TransactionResults/types.d.ts [hunk 4]
```diff
@@ -5,6 +5,8 @@ import {
   FioAddressWithPubAddresses,
   LinkActionResult,
 } from '../../../types';
+import { InfoBadgeProps } from '../../InfoBadge/InfoBadge';
+import SubmitButton from '../SubmitButton/SubmitButton';
```
Comment (andreyvEze) [inline client/src/components/common/TransactionResults/types.d.ts:9]: move above types import

File: client/src/components/common/TransactionResults/types.d.ts [hunk 5]
```diff
@@ -25,6 +27,9 @@ export type ResultsProps = {
   titleTo?: string;
   titleFrom?: string;
   titleAmount?: string;
+  labelAmount?: string;
+  infoBadges?: InfoBadgeProps[];
+  additionalSubmitButtonProps?: SubmitButton;
```
Comment (andreyvEze) [inline client/src/components/common/TransactionResults/types.d.ts:32]: we should set this props only for staking results component

File: client/src/pages/UnstakeTokensPage/UnstakeTokensPage.tsx [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+import UnstakeTokensForm from './components/UnstakeTokensForm';
+import UnstakeTokensEdgeWallet from './components/UnstakeTokensEdgeWallet';
+import StakeTokensResults from '../../components/common/TransactionResults/components/StakeTokensResults';
+
+import { putParamsToUrl } from '../../utils';
+import { convertFioPrices } from '../../util/prices';
+import { useFioAddresses } from '../../util/hooks';
+
+import { DEFAULT_ACTION_FEE_AMOUNT, TrxResponse } from '../../api/fio';
+import { ResultsData } from '../../components/common/TransactionResults/types';
```
Comment (andreyvEze) [inline client/src/pages/UnstakeTokensPage/UnstakeTokensPage.tsx:14]: move to types import

File: client/src/pages/UnstakeTokensPage/UnstakeTokensPage.tsx [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+
+import { ROUTES } from '../../constants/routes';
+import { WALLET_CREATED_FROM } from '../../constants/common';
+
+import { ContainerProps, StakeTokensValues, InitialValues } from './types';
+
+import classes from './styles/UnstakeTokensPage.module.scss';
+import { BADGE_TYPES } from '../../components/Badge/Badge';
```
Comment (andreyvEze) [inline client/src/pages/UnstakeTokensPage/UnstakeTokensPage.tsx:22]: move to const imports

**Review by andreyvEze — CHANGES_REQUESTED** [review]
check comments

File: client/src/components/Input/StakeAmountInput.tsx (hunk 1 above)
Comment (NikolasStelmakh) [reply client/src/components/Input/StakeAmountInput.tsx:46]: it was in previous commits, but due to futher Number() manipulations in component it was refactored in fixup to different types

**Pull request merged**